src/main/java/com/almightyalpaca/discord/jdabutler/commands/commands/HelpCommand.java

```diff
@@ -0,0 +1,42 @@
+package com.almightyalpaca.discord.jdabutler.commands.commands;
+
+import com.almightyalpaca.discord.jdabutler.Bot;
+import com.almightyalpaca.discord.jdabutler.EmbedUtil;
+import com.almightyalpaca.discord.jdabutler.commands.Command;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.MessageBuilder;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.stream.Collectors;
+
+public class HelpCommand implements Command {
+    @Override
+    public void dispatch(String[] args, User sender, TextChannel channel, Message message, String content) {
+        EmbedBuilder builder = new EmbedBuilder();
+        EmbedUtil.setColor(builder);
+        String help = Bot.dispatcher.getCommands().stream()
+                .filter(c -> c.getHelp() != null)
+                .map(c -> String.format("%s%s - %s", Bot.config.getString("prefix", ""), c.getName().toLowerCase(), c.getHelp()))
+                .collect(Collectors.joining("\n"));
+        builder.setAuthor("Help Command", null, Bot.jda.getSelfUser().getEffectiveAvatarUrl());
+        builder.setDescription(help);
+        builder.setFooter(String.format("Requested on %s by %s",
```

Comment:
I'm not a fan of useless footers, why not put the requester as `author` instead of the `"Help Command"`.
This is redundant clutter in my opinion.